app/assets/stylesheets/ltags/DevCommentTag.scss

```diff
@@ -61,11 +62,13 @@
       width:100px;
       a{
         color:$medium-gray;
+        color: var(--theme-secondary-color, $medium-gray);
       }
     }
 
     a {
       color: lighten($dark-gray,3%);
+      color: var(--theme-secondary-color, $dark-gray);
```

Comment:
Don't we want to keep the lighten function?

Reply:
@Glennmen 
as for the changes I have made, the `lighten()` function is not required in the `dark mode` as far as , I have assumed that it might be of use in the default mode.

Rest is upto the mods I guess.

Reply:
You might be right about the dark mode not needing it (DEV team should decide about this), but the fallback variable is not having the lighten function anymore so the color will not be the same anymore when dark mode is disabled.

Reply:
Will make the changes 👍